GSG-K3/urban-calendar
# Component/calendar

### Content
- Add Calendar Component to the Stepper.
- Add Appointment Time field to Formik as radio button, give it initial values, and validate it.
- The user contact information and reservation data will be stored on the database when he/she selects a specific date and time from the calendar component.
---
### UI

![Screenshot from 2020-07-14 15-41-05](https://user-images.githubusercontent.com/54964739/87427955-42aeea00-c5ea-11ea-8372-dd5ae7fc2722.png)

---
### Testing
on terminal use the following commands:
- pull origin component/calendar
- npm run dev
- go to http://localhost:3000/reservation on your browser.
- After you fill all the information and submitting the form you can check your database using this SQL query:
```
$ select * from customer_info;

```
P.S. If you don't have a local database connected with this project you can follow the steps mentioned here:
https://github.com/GSG-K3/urban-calendar/blob/master/README.md
### Next Step
   - [x]  Check if the user answer of the COVID19 question was yes, then we should disable 14 days from today on the calendar.
   - [x]  Disable Sundays and Tuesdays as off dates of the store & public holidays
   - [x]  Create labels or headers to tell the user, please choose a date for the calendar then please choose a time for the time slots.    
   - [x] Changing Radio Buttons style.
   - [x]  Call an API to get the available time slots for any selected day on the calendar.
